Pinta/Pads/HistoryPad.cs

```diff
@@ -33,27 +33,21 @@ namespace Pinta
 {
 	public class HistoryPad : IDockPad
 	{
-		public void Initialize (Dock workspace, Application app, GLib.Menu padMenu)
+		public void Initialize (DockFrame workspace, Application app, GLib.Menu padMenu)
 		{
 			var history = new HistoryTreeView ();
-			DockItem history_item = new DockItem(history, "History")
-			{
-				Label = Translations.GetString("History")
-			};
+			DockItem history_item = workspace.AddItem ("History");
+			DockItemToolbar history_tb = history_item.GetToolbar (PositionType.Bottom);
 
-			// TODO-GTK3 (docking)
-#if false
+			history_item.Label = ("History");
 			history_item.DefaultLocation = "Images/Bottom";
-			history_item.Icon = Gtk.IconTheme.Default.LoadIcon(Resources.Icons.LayerDuplicate, 16);
-			history_item.DefaultWidth = 100;
+			history_item.Content = history;
+			history_item.Icon = PintaCore.Resources.GetIcon ("Menu.Layers.DuplicateLayer.png");
```

Comment:
In the GTK3 build we now use themed icons, so I think this should be `Gtk.IconTheme.Default.LoadIcon(Resources.Icons.LayerDuplicate, 16);` from the deleted line above?